_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -141,55 +141,58 @@ $shadow: transparentize(palette(grayscale, black-t), .5);
 // ----------------------------
 // #TYPOGRAPHY
 // ----------------------------
+
+$ux-pattern-library-font-path: '../fonts' !default;
```

Comment:
Would you mind changing this variable name to match some of the conventions used for the rest of our typography variables? Something like ``$font-path`` seems more appropriate.